### PR TITLE
Make buildifier sort named arguments of all calls.

### PR DESCRIPTION
**Pull request by fourdman**
Currently buildifer sorts named arguments only when the called function is a literal. This change allows the called function to be any expression, e.g. `foo.bar(a=1,b=2)` or `foo().bar(a=1,b=2)`.

### CODE REVIEW

File: build/rewrite.go [hunk 1]
```diff
@@ -288,7 +288,7 @@ func sortCallArgs(f *File) {
 		}
 		rule := callName(call)
 		if rule == "" {
-			return
+			rule = "<complex rule kind>"
```
Comment (vladmos) [inline build/rewrite.go:291]: I'd also extract the names from DotExpr nodes (such as `foo.bar(...)`). This way we can have custom sorting rules for them in `tables.NamePriority` too.

Comment (fourdman) [reply build/rewrite.go:288]: Done (updated callName()).

**Review by vladmos — APPROVED** [review]
Thanks. I'll be on vacation next week, will merge and import the change after that.

**Comment by laurentlb** [comment]
fyi - this change should be mentioned in the release notes.

**Pull request merged**